Makefile

```diff
@@ -1,19 +1,12 @@
 DOCKER_REPO ?= icecave/honeycomb
 DOCKER_TAG  ?= dev
 
-REQ += $(patsubst res/assets/%,artifacts/assets/%.go, $(wildcard res/assets/*))
-DOCKER_REQ += artifacts/cacert.pem
+GENERATED_FILES += $(patsubst res/assets/%,artifacts/assets/%.go, $(wildcard res/assets/*))
+DOCKER_BUILD_REQ += artifacts/cacert.pem
 
-GIT_HASH ?= $(shell git show -s --format=%h)
-GIT_TAG ?= $(shell git tag -l --merged $(GIT_HASH) | tail -n1)
-APP_VERSION ?= $(if $(TRAVIS_TAG),$(TRAVIS_TAG),$(if $(GIT_TAG),$(GIT_TAG),$(GIT_HASH)))
-APP_DATE ?= $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Alas poor `APP_VERSION`, I knew thee well :)

Reply:
It's still there though, kinda!

- https://github.com/make-files/pkg-go/blob/v1/Makefile#L27
- https://github.com/make-files/lib-core/blob/v1/include/common.mk#L14